File: module_media/src/main/java/com/wangzhumo/app/module/media/targets/task3_2/ITextureRenderer.java
package com.wangzhumo.app.module.media.targets.task3_2;

import android.graphics.SurfaceTexture;

/**
 * @anchor: andy
 * @date: 2018-11-11
 * @description:
 */
public interface ITextureRenderer {

    void onSurfaceCreated();

    void onSurfaceChanged(int width, int height);

    void onDrawFrame(SurfaceTexture surfaceTexture);

}

File: module_media/src/main/java/com/wangzhumo/app/module/media/opengl/camera/ITextureRenderer.kt
package com.wangzhumo.app.module.media.opengl.camera

import android.graphics.SurfaceTexture


/**
 * If you have any questions, you can contact by email {wangzhumoo@gmail.com}
 *
 * @author 王诛魔 2019-11-22  20:54
 *
 * Renderer要实现的方法.
 */
interface ITextureRenderer {

    fun onSurfaceCreated()

    fun onSurfaceChanged(width: Int, height: Int)

    fun onDrawFrame(surfaceTexture: SurfaceTexture?)
}